packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -193,10 +195,11 @@ export class WebsocketConnector {
                             port: this.selectedPort!,
                             host: this.host,
                             tls: this.config.serverEnableTls,
-                            selfSigned
+                            selfSigned,
+                            version
                         }
                         if (!this.abortController.signal.aborted) {
-                            return await sendConnectivityRequest(connectivityRequest, entryPoint)
+                            return await sendConnectivityRequest(connectivityRequest, entryPoint, version)
```

Comment:
Could maybe rename `version` as `localVersion`  when importing so that it is clear when we use it?